service/matching/matchingEngine_test.go

```diff
@@ -255,10 +255,12 @@ func (s *matchingEngineSuite) AddTasksTest(taskType int) {
 			err = s.matchingEngine.AddActivityTask(&addRequest)
 		} else {
 			addRequest := matching.AddDecisionTaskRequest{
-				DomainUUID: common.StringPtr(domainID),
-				Execution:  &execution,
-				ScheduleId: &scheduleID,
-				TaskList:   taskList}
+				DomainUUID:                    common.StringPtr(domainID),
+				Execution:                     &execution,
+				ScheduleId:                    &scheduleID,
+				TaskList:                      taskList,
+				ScheduleToStartTimeoutSeconds: common.Int32Ptr(1),
```

Comment:
This could cause test flakiness if we are reading these tasks back.  Do we use the same timeout for ActivityTask?

Reply:
I do use same timeout as ActivityTask, should be fine.